src/FSharpx.Extras/paket.template

```diff
@@ -21,5 +21,6 @@ summary
     FSharpx.Extras implements general functional constructs on top of the F# core library. Its main target is F# but it aims to be compatible with all .NET languages wherever possible.
 description
     FSharpx.Extras implements general functional constructs on top of the F# core library. Its main target is F# but it aims to be compatible with all .NET languages wherever possible.
-
+files
+    src/FSharpx.Extras/bin/Release/net45/* ==> lib/net45
```

Comment:
you really need that? you can generate with `dotnet pack`. paket support that (will use paket refs)